.github/workflows/web.yml

```
@@ -56,7 +56,9 @@ jobs:
     outputs:
       has_frontend_changes: ${{ steps.filter.outputs.has_frontend_changes }}
       has_backend_changes: ${{ steps.filter.outputs.has_backend_changes }}
-      has_features_changes: ${{ steps.filter.outputs.has_features_changes }}
+      has_test_changes: ${{ steps.filter.outputs.has_test_changes }}
+      trigger_delivery: ${{ steps.changes_trigger_checks.outputs.trigger_delivery }}
```

actionlint

property "changes_trigger_checks" is not defined in object type {filter: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)